extensions/notebook/src/book/bookTreeItem.ts

```diff
@@ -36,6 +22,7 @@ export interface BookTreeItemFormat {
 	version?: BookVersion;
 	parent?: BookTreeItem;
 	hasChildren?: boolean;
+	hierarchyId?: string;
```

Comment:
Doc comment please, this isn't really clear as to what exactly a "hierarchy id" would be used for